src/Preflight/Preflight.php

```diff
@@ -255,6 +255,11 @@ public function preflight($argv): array
         // Now that we know the value, set debug flag.
         $this->logger()->setDebug($this->preflightArgs->get(PreflightArgs::DEBUG, false));
 
+        // Give hint if a developer might be trying to debug Drush.
+        if (extension_loaded('xdebug')) {
+            $this->logger()->log(strtr('If you are trying to perform step debugging of Drush, see !url', ['!url' => 'https://www.drush.org/latest/usage/#xdebug']));
```

Comment:
*If* we're going to keep `--xdebug`, does it make sense to mention it directly here? Maybe something like
> If you are trying to perform step debugging of Drush use the --xdebug option, or see !url for more options.

Reply:
I improved the wording and fixed the url. I opted not to mention --xdebug there because the dev needs to decide which approach suits her better. Thats best done by reading the docs.